File: Latihan 2/bahasaphp/Product.php
<?php 
 
 //parent class
 class Product{
	
	//atribut private
	private $price;
	private $id;
	
	//constructor
	function __construct(){
	}
		
	//proses get dan set
	function setPrice($price){
		$this->price = $price;
	}
		
	function getPrice(){
		return $this->price;
	}
		
	function setId($id){
		$this->id = $id;
	}
		
	function getId(){
		return $this->id;
	}
	
	//destructor
	 function __destruct(){
	 }
}
?>

File: Latihan 2/bahasac++/Product.cpp
class Product{ //base class
	
	//atribut
	private:
	string price;
	string id;

	//constructor
	public:
	
	Product(){
	}
	
	//proses get dan set
	void setPrice(string price){
		this->price = price;
	}
	
	string getPrice(){
		return this->price;
	}
	
	void setId(string id){
		this->id = id;
	}
	
	string getId(){
		return this->id;
	}
	
	//destructor
	~Product(){
	}
};
	
	